pkg/controllers/binding/common.go

```diff
@@ -168,6 +170,7 @@ func mergeLabel(workload *unstructured.Unstructured, binding metav1.Object, scop
 
 func mergeAnnotations(workload *unstructured.Unstructured, binding metav1.Object, scope apiextensionsv1.ResourceScope) map[string]string {
 	annotations := make(map[string]string)
+	util.MergeAnnotation(workload, workv1alpha2.ResourceTemplateGenerationAnnotationKey, fmt.Sprintf("%v", workload.GetGeneration()))
```

Comment:
move to work mutating webhook.
https://github.com/karmada-io/karmada/blob/2ed4d77a0617898eced66f9eb5727d7c4d8222b8/pkg/webhook/work/mutating.go#L97-L98

Reply:
use `strconv.FormatInt(workload.GetGeneration(), 10)` instead of `fmt.Sprintf("%v", workload.GetGeneration())`.

Reply:
done